src/elements/content-sidebar/Sidebar.js

```diff
@@ -175,6 +182,46 @@ class Sidebar extends React.Component<Props, State> {
         }
     }
 
+    /**
+     * Handle keyboard interaction
+     *
+     * @param {Array<string>} tablist - Contains the pathnames of tabs
+     * @param {SyntheticKeyboardEvent} event - Keyboard event
+     */
+    onKeyDown = (tablist: Array<string>) => (event: SyntheticKeyboardEvent<>): void => {
```

Comment:
The most common convention uses the prefix `on` for event-oriented prop names and `handle` for their values:

```
onKeyDown={handleKeyDown}
```